server/database/config/db_connection.js

```diff
@@ -0,0 +1,27 @@
+const { Pool } = require('pg');
+require('dotenv').config();
+const url = require('url');
+
+let DB_URL = '';
+if (process.env.NODE_ENV === 'test') {
```

Comment:
``` NODE_ENV ``` should be ``` development ``` when we in the development stage. 
``` NODE_ENV ``` should be ``` production ``` when we in the production stage. 
``` NODE_ENV ``` should be ``` testing ``` when we in the testing stage.